biceps-model/build.gradle.kts

```diff
@@ -0,0 +1,47 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ */
+
+
+plugins {
+    id("com.draeger.medical.java-conventions")
+    id("org.somda.sdc.xjc")
+}
+
+val jaxb: Configuration by configurations.creating
+val schemaDir = "src/main"
+
+
+dependencies {
+    api(libs.org.glassfish.jaxb.jaxb.core)
+    api(libs.org.glassfish.jaxb.jaxb.runtime)
+    api(libs.jakarta.xml.bind.jakarta.xml.bind.api)
+    api(libs.io.github.threeten.jaxb.threeten.jaxb.core)
+    api(libs.org.jvnet.jaxb.jaxb.plugins)
+    api(libs.org.checkerframework.checker.qual)
+    api(libs.org.jetbrains.annotations)
+
+    testImplementation(libs.org.junit.jupiter.junit.jupiter.api)
+    testImplementation(libs.org.junit.jupiter.junit.jupiter.engine)
+
+    jaxb(libs.org.jetbrains.annotations)
+    jaxb(libs.org.jvnet.jaxb.jaxb.plugin.annotate)
+}
+
+description = "SDCri is a set of Java libraries that implements a network communication framework conforming " +
+    "with the IEEE 11073 SDC specifications. This project implements the model for IEEE 11073-10207."
+
+xjc {
+    jaxbClasspath = jaxb
+    schemaLocation = layout.projectDirectory.dir(schemaDir)
+    args = listOf(
+        "-Xannotate",
```

Comment:
This flag was also not enabled by our model previously. Now, it doesn't do anything, as we have no annotations in the .xjb file, but still.